book/src/usage.md

```diff
@@ -1,22 +1,44 @@
-# Usage
+# Using Helix
+
+<!--toc:start-->
+- [Using Helix](#using-helix)
+  - [Registers](#registers)
+    - [User-defined Registers](#user-defined-registers)
+    - [Built-in Registers](#built-in-registers)
+  - [Surround](#surround)
+  - [Moving the Selection with Syntax-tree Motions](#moving-the-selection-with-syntax-tree-motions)
+  - [Selecting and Manipulating Text with Textobjects](#selecting-and-manipulating-text-with-textobjects)
+  - [Navigating Using Tree-sitter Textobjects](#navigating-using-tree-sitter-textobjects)
+  <!--toc:end-->
+
+For a full interactive introduction to Helix, refer to the
+[tutor](https://github.com/helix-editor/helix/blob/master/runtime/tutor) which
+can be accessed via the command `hx --tutor` or `:tutor`.
+
+> 💡 Currently, not all functionality is fully documented, please refer to the
+> [key mappings](./keymap.md) list.
 
-(Currently not fully documented, see the [keymappings](./keymap.md) list for more.)
+## Registers
 
-See [tutor](https://github.com/helix-editor/helix/blob/master/runtime/tutor) (accessible via `hx --tutor` or `:tutor`) for a vimtutor-like introduction.
+In Helix, registers are storage locations for text and other data, such as the
+result of a search. Registers can be used to cut, copy, and paste text, similar
+to the clipboard in other text editors. Usage is similar to Vim, with `"` being
+used to select a register.
 
-## Registers
+### User-defined Registers
 
-Vim-like registers can be used to yank and store text to be pasted later. Usage is similar, with `"` being used to select a register:
+Helix allows you to create your own named registers for storing text, for
+example:
 
 - `"ay` - Yank the current selection to register `a`.
 - `"op` - Paste the text in register `o` after the selection.
 
-If there is a selected register before invoking a change or delete command, the selection will be stored in the register and the action will be carried out:
+If a register is selected before invoking a change or delete command, the selection will be stored in the register and the action will be carried out:
 
 - `"hc` - Store the selection in register `h` and then change it (delete and enter insert mode).
 - `"md` - Store the selection in register `m` and delete it.
 
-### Special Registers
+### Built-in Registers
```

Comment:
Why the rename? These are called "special registers" in kakoune and vim so we were sticking to the same historical naming convention.

Reply:
I didn't know that! I thought `special` was just a word that someone had come up with that didn't mean much, my mistake!